src/shared/utils.js

```diff
@@ -4,6 +4,12 @@ function convertToSafeString(string){
 			.toLowerCase();
 }//convertToSafeString
 
+function convertToHyphenatedString(string){
+	return string.replace(/\W+/g, '-') // remove non-alphanumeric & replace with -s
+			.toLowerCase()
+}//convertToHyphenatedString
```

Comment:
I don't think we need a separate function for this. If we just change `convertToSafeString` to be:
```
function convertToSafeString(string, separator="_"){
    return string.replace(/\W/g, "").replace(/ /g, separator).toLowerCase();
}
```
then we can just do convertToSafeString("whatever episode #2", "-"); if we want to separate with hyphens instead of underscores.

Reply:
Done